faynoSync_test.go

```
@@ -4986,10 +5045,130 @@ func TestCheckVersion(t *testing.T) {
+			cdnResponseURL = cdnResponseURL.JoinPath(
+				"responses",
+				"admin",
+				"public testapp",
+				combo.ChannelName,
+				combo.Platform,
+				combo.Arch,
+				combo.AppVersion+".json",
+			)
+
+			httpClient := &http.Client{Timeout: 10 * time.Second}
+			resp, err := httpClient.Get(cdnResponseURL.String())
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)